components/inferno/traduction/README.de.md

```diff
@@ -0,0 +1,193 @@
+
+## Installation
+
+```shell
+
+npm install inferno-particles
+
+```
+
+oder
+
+```shell
+
+yarn add inferno-particles
+
+```
+
+## Bedienungsanleitung
+
+### Code
+
+Beispiel:
+
+```javascript
+import Particles from "inferno-particles";
+
+class App extends Component {
+  render() {
+    return (
+      <Particles
+        id="tsparticles"
+        params={{
+          background: {
+            color: {
+              value: "#0d47a1",
+            },
+          },
+
+          fpsLimit: 60,
+
+          interactivity: {
+            detectsOn: "canvas",
+
+            events: {
+              onClick: {
+                enable: true,
+
+                mode: "push",
+              },
+
+              onHover: {
+                enable: true,
+
+                mode: "repulse",
+              },
+
+              resize: true,
+            },
+
+            modes: {
+              bubble: {
+                distance: 400,
+
+                duration: 2,
+
+                opacity: 0.8,
+
+                size: 40,
+              },
+
+              push: {
+                quantity: 4,
+              },
+
+              repulse: {
+                distance: 200,
+
+                duration: 0.4,
+              },
+            },
+          },
+
+          particles: {
+            color: {
+              value: "#ffffff",
+            },
+
+            links: {
+              color: "#ffffff",
+
+              distance: 150,
+
+              enable: true,
+
+              opacity: 0.5,
+
+              width: 1,
+            },
+
+            collisions: {
+              enable: true,
+            },
+
+            move: {
+              direction: "none",
+
+              enable: true,
+
+              outMode: "bounce",
+
+              random: false,
+
+              speed: 6,
+
+              straight: false,
+            },
+
+            number: {
+              density: {
+                enable: true,
+
+                value_area: 800,
+              },
+
+              value: 80,
+            },
+
+            opacity: {
+              value: 0.5,
+            },
+
+            shape: {
+              type: "circle",
+            },
+
+            size: {
+              random: true,
+
+              value: 5,
+            },
+          },
+
+          detectRetina: true,
+        }}
+      />
+    );
+  }
+}
+```
+
+### Props
+
+| Prop | Typ | Definition |
+
+| --------------- | ------ | -------------------------------------------------------------------------------------------------------------------- |
+
+| width | string | Die Breite der canvas. |
+
+| height | string | Die Höhe der canvas. |
+
+| options | object | Die Optionen der Partikelinstanz. |
+
+| style | object | das Design des canvas-Elements |
+
+| className | string | Der Klassenname des canvas-wrappers |
+
+| canvasClassName | string | Der Klassenname des canvas. |
+
+| container | object | Die Instanz des [Partikel-Containers]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)